.fastlane/Fastfile

```diff
@@ -0,0 +1,133 @@
+    end
+    project.save
+  end
+
+  private_lane :build_app_production do
+    match(
+      type: "appstore",
+      git_url: ENV['CERTIFICATES_REPO'],
+      app_identifier: [ "cc.newspring.newspringapp", "com.subsplashstudio31.NewSpring-Church" ],
+      readonly: true,
+      username: ENV["APPLE_ID"],
+    )
+
+    increment_build_number(
+      xcodeproj: "#{ENV['METEOR_OUTPUT_ABSOLUTE']}/ios/project/#{ENV['XCODE_SCHEME_NAME']}.xcodeproj",
+      build_number: ENV['APP_BUILD'],
+    )
+
+    increment_version_number(
+      xcodeproj: "#{ENV['METEOR_OUTPUT_ABSOLUTE']}/ios/project/#{ENV['XCODE_SCHEME_NAME']}.xcodeproj",
+      version_number: ENV['APP_VERSION'],
+    )
+
+    update_provisioning_profile_specifier(
+      project: "#{ENV['METEOR_OUTPUT_ABSOLUTE']}/ios/project/#{ENV['XCODE_SCHEME_NAME']}.xcodeproj",
+      prov_name: ENV["PROVISIONING_PROFILE_SPECIFIER"],
+      build_configuration: "Release",
+      target_filter: ENV['XCODE_SCHEME_NAME'],
+    )
+
+    set_info_plist_value(
+      path: "#{ENV['METEOR_OUTPUT_ABSOLUTE']}/ios/project/#{ENV['XCODE_SCHEME_NAME']}/#{ENV['XCODE_SCHEME_NAME']}-Info.plist",
+      key: "NSCameraUsageDescription",
+      value: "Profile Upload",
+    )
+
+    set_info_plist_value(
+      path: "#{ENV['METEOR_OUTPUT_ABSOLUTE']}/ios/project/#{ENV['XCODE_SCHEME_NAME']}/#{ENV['XCODE_SCHEME_NAME']}-Info.plist",
+      key: "NSPhotoLibraryUsageDescription",
+      value: "Profile Upload",
+    )
+
+    set_info_plist_value(
+      path: "#{ENV['METEOR_OUTPUT_ABSOLUTE']}/ios/project/#{ENV['XCODE_SCHEME_NAME']}/#{ENV['XCODE_SCHEME_NAME']}-Info.plist",
+      key: "NSLocationWhenInUseUsageDescription",
+      value: "Group Finder",
+    )
+
+    gym(
+      project: "#{ENV['METEOR_OUTPUT_ABSOLUTE']}/ios/project/#{ENV['XCODE_SCHEME_NAME']}.xcodeproj",
+      scheme: ENV['XCODE_SCHEME_NAME'],
+      clean: true,
+      include_bitcode: false,
+      export_team_id: ENV['TEAM_ID'],
+      codesigning_identity: ENV["CODESIGNING_IDENTITY"],
+    )
+
+  end
+
+  desc "Just build the app"
+  lane :build do
+    build_app_production
+  end
+
+  desc "Install App Store Certificates"
+  lane :certificates do
+    match(
+      type: "appstore",
+      git_url: ENV['CERTIFICATES_REPO'],
+      app_identifier: [ "cc.newspring.newspringapp", "com.subsplashstudio31.NewSpring-Church" ],
+      username: ENV["APPLE_ID"],
+    )
+  end
+
+  desc "Submit a new Beta Build to Apple TestFlight"
+  desc "This will also make sure the profile is up to date"
+  lane :beta do
+    build_app_production
+
+    pilot(
+      username: ENV['APPLE_ID'],
+      skip_waiting_for_build_processing: true,
+      app_identifier: "cc.newspring.newspringapp",
+      changelog: ENV['RELEASE_NOTES'],
+    )
+  end
+
+  desc "Deploy a new version to the App Store"
+  lane :production do
+    build_app_production
+
+    deliver(
+      username: ENV['APPLE_ID'],
+      force: true,
+      app_identifier: "com.subsplashstudio31.NewSpring-Church",
```

Comment:
the `deliver` action doesn't have a changelog parameter like pilot does. It does have metadata but i think we should investigate that further during another sprint.